M6-Backend/FacilityInterfaces/interfaceAddFacility.js

```diff
@@ -0,0 +1,42 @@
+
+function interfaceAddFacility(Info){
+    var newId = Info.id;
+    var title = Info.title;
+    var description = Info.description;
+    var long = parseFloat(Info.long);
+    var lat = parseFloat(Info.lat);
+    var type = Info.type;  
```

Comment:
Codacy has a fix for the issue: [Replace `··var·type·=·Info.type;··` with `var·type·=·Info.type;`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
  var type = Info.type;
```